hbase/Dockerfile

```diff
@@ -41,43 +42,22 @@ WORKDIR /stackable
 RUN curl --fail -L "https://repo.stackable.tech/repository/packages/hbase/hbase-${PRODUCT}-src.tar.gz" | tar -xzC .
 RUN mv hbase-${PRODUCT} hbase-${PRODUCT}-src
+COPY --chown=stackable:stackable hbase/stackable/patches /stackable/patches
 RUN chmod +x patches/apply_patches.sh
 RUN patches/apply_patches.sh ${PRODUCT}
 RUN cd /stackable/hbase-${PRODUCT}-src/ && \
-    mvn -Dhadoop.profile=3.0 -Dhadoop-three.version=${HADOOP_M2} clean install -DskipTests && \
-    mvn -Dhadoop.profile=3.0 -Dhadoop-three.version=${HADOOP_M2} install assembly:single -DskipTests -Dcheckstyle.skip=true -Prelease && \
+    mvn -Dhadoop.profile=3.0 -Dhadoop-three.version=${HADOOP} clean install -DskipTests && \
+    mvn -Dhadoop.profile=3.0 -Dhadoop-three.version=${HADOOP} install assembly:single -DskipTests -Dcheckstyle.skip=true -Prelease && \
     tar -xzf hbase-assembly/target/hbase-${PRODUCT}-bin.tar.gz -C /stackable/ && \
     rm -rf /stackable/hbase-${PRODUCT}-src
 
-# Build the hbase-operator-tools
-RUN curl --fail -L "https://repo.stackable.tech/repository/packages/hbase-operator-tools/hbase-operator-tools-${HBASE_OPERATOR_TOOLS}-src.tar.gz" | tar -xzC . && \
-    mv hbase-operator-tools-${HBASE_OPERATOR_TOOLS} hbase-operator-tools-${HBASE_OPERATOR_TOOLS}-src && \
-    mvn \
-        -Dhbase.version=${PRODUCT} \
-        -Dhbase-thirdparty.version=${HBASE_THIRDPARTY} \
-        -DskipTests \
-        -fhbase-operator-tools-${HBASE_OPERATOR_TOOLS}-src \
-        package assembly:single && \
-    tar -xz \
-        -f hbase-operator-tools-${HBASE_OPERATOR_TOOLS}-src/hbase-operator-tools-assembly/target/hbase-operator-tools-${HBASE_OPERATOR_TOOLS}-bin.tar.gz \
-        -C /stackable/ && \
-    rm -rf /stackable/hbase-operator-tools-${HBASE_OPERATOR_TOOLS}-src
-
-# Resolve paths in bin/hbck2
-RUN mv bin/hbck2 bin/hbck2.env && \
-    # The variable names are intentionally passed to envsubst in single-quotes,
-    # so that they are not expanded. Disabling ShellCheck rules in a Dockerfile
-    # does not work, so please ignore the according warning (SC2016).
-    envsubst '${PRODUCT}:${HBASE_OPERATOR_TOOLS}' < bin/hbck2.env > bin/hbck2 && \
-    chmod +x bin/hbck2 && \
-    rm bin/hbck2.env
-
 # The symlink from JMX Exporter 0.16.1 to the versionless link exists because old HBase Operators (up until and including 23.7) used to hardcode
 # the version of JMX Exporter like this: "-javaagent:/stackable/jmx/jmx_prometheus_javaagent-0.16.1.jar"
 # This is a TEMPORARY fix which means that we can keep the hardcoded path in HBase operator FOR NOW as it will still point to a newer version of JMX Exporter, despite the "0.16.1" in the name.
 # At the same time a new HBase Operator will still work with older images which do not have the symlink to the versionless jar.
 # After one of our next releases (23.11 or 24.x) we should update the operator to point at the non-versioned symlink (jmx_prometheus_javaagent.jar)
```

Comment:
The HBase operator uses the non-versioned symlink since SDP 23.11 (https://github.com/stackabletech/hbase-operator/pull/403/files#diff-b10253ea28ef51076309f375e97cd5e2795500b0e07c5326c413055bffcb4a85).

I removed the versioned symlink and updated the changelog (d8ba5d899988414f08baaf8948a82dd5dab35909).

Reply:
Thanks!